### DISCUSSION TITLE
Wie kann ich das Grid Profil ändern

### DISCUSSION THREAD

**Discussion by Marc2805** [tbnobody/OpenDTU#1625] [General]
Hallo,
würde gerne das grid profil meines Hoymiles HM 1500 wechseln von XX - EN 50549-1:2019 auf DE VDE-AR-N 4105:2018
Auf meinen WR kann ich nur über opendtu zugreifen nicht über die s-miles cloud.
gibt es eine möglichkeit das grid profil in opendtu zu ändern.

mfg Marc

**Comment by D3R-ST3FAN** [comment]
Nein, das geht im Moment (noch) nicht mit OpenDTU. Hierfür wird die original DTU + s-miles Account benötigt.

**Reply by 4c2** [reply]
Hallo, danke für die Info, das interessiert mich auch. 
Git es eine (zeitliche) Planung, dieses Feature in die OpenDTU rein zu nehmen?

**Comment by xX-nichtlachen-Xx** [comment]
Implemented here:
https://github.com/xX-nichtlachen-Xx/OpenDTU/tree/GridProfileSelector_Editor
